src/redux/features/Plantings/plantingsSelectors.ts

```diff
@@ -1,11 +1,47 @@
 import { RootState } from 'src/redux/rootReducer';
+import { selectPlantingSites } from '../tracking/trackingSelectors';
+import { createSelector } from '@reduxjs/toolkit';
 
 export const selectPlantings = (state: RootState) => state.plantings?.plantings;
 
+export const selectTotalPlantsBySubzone = (state: RootState) => {
+  const plantingsBySubzone: Record<number, { totalPlants: number }> = {};
+  state.plantings?.plantings?.forEach((planting) => {
+    if (plantingsBySubzone[Number(planting.plantingSubzone.id)]) {
```

Comment:
thanks for the update, good to sum them up.
can you extract out Number(planting.plantingSubzone.id) into a constant?
also if you can, change this into a reduce pattern (we have been doing that mostly)